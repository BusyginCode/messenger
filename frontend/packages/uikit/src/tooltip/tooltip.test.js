import React from 'react'
import { shallow } from 'enzyme'
import Tooltip from './tooltip'

describe('uikit/Tooltip', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<Tooltip id={1} />)
    expect(wrapper).toMatchSnapshot()
  })
})
