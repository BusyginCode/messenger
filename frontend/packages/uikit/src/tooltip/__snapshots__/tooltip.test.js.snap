// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`uikit/Tooltip should render correctly 1`] = `
<Tooltip
  align={Object {}}
  arrowContent={null}
  className="tooltip"
  destroyTooltipOnHide={false}
  id={1}
  mouseEnterDelay={0}
  mouseLeaveDelay={0.1}
  onPopupAlign={[Function]}
  onVisibleChange={[Function]}
  placement="right"
  prefixCls="dt-tooltip"
  trigger={
    Array [
      "hover",
    ]
  }
>
  <ChildWrapper
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    style={Object {}}
  />
</Tooltip>
`;
